tests/test_smab.py

```
@@ -86,28 +88,38 @@ class ModelTestConfig:
         if len(model_types) < len(action_ids):
             indices = np.random.randint(0, len(model_types), len(action_ids))
             model_types = [model_types[i] for i in indices]
         if all(model in [BetaCC, ZoomingCC, BetaMOCC] for model in model_types):
             # Generate random costs
-            costs = costs.draw(cost_strategy(n_actions=len(action_ids)))
+            drawn_values = values.draw(value_strategy(n_actions=len(action_ids)))
             costs = [
-                cost if model_type in [BetaCC, BetaMOCC] else partial(_quantitative_cost, cost=cost)
-                for cost, model_type in zip(costs, model_types)
+                val if model_type in [BetaCC, BetaMOCC] else partial(_quantitative_callable, value=val)
+                for val, model_type in zip(drawn_values, model_types)
```

Ruff (0.15.15)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)